### ISSUE TITLE
Give paragraphs a max-width

### ISSUE THREAD

**Issue by alexanderwallin** [webpack/webpack.js.org#191]
Great to see new docs in the makings!

One thing I think the old and new docs alike are suffering from is readability, the biggest problem being the number of characters per line (120+ on my 13" MacBook).

Attaching an example of paragraph width constraits and use of system font.

**Before**

<img width="956" alt="webpack1" src="https://cloud.githubusercontent.com/assets/6952455/18833352/1d7da8a4-83e8-11e6-8e57-5c5b32ae841a.png">

**After**

```css
p {
  max-width: 35em;
  line-height: 1.5;
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
}
```

<img width="958" alt="webpack2" src="https://cloud.githubusercontent.com/assets/6952455/18833351/1cce74a6-83e8-11e6-8826-62b5c02ebc66.png">

**Comment by gregvenech** [comment]
Hi @alexanderwallin, I definitely agree that we should have an optimal width for reading. However I just double checked the deployed version and found that even at our widest screen width we're under 960px (currently) for the content section. We are planning to bring that down for smaller screens, see #183, but isn't 960px the recommended max for readability?

Either way I'm fine with bringing it down a bit but I'd rather take the max screen size, see [_screens.scss], than individual paragraphs.

EDIT: After a little more research, I guess you're right that typically these days 560 - 720px (at a font-size of 16) is the recommended norm.

**Comment by alexanderwallin** [comment]
I think the key metric is the number of characters per line, where anything between ~45 and ~95 seem to be recommended. 720px results in about 120 characters per line on my machine.

Thanks for the quick reply!

**Comment by gregvenech** [comment]
No problem, I'll play around with it and tonight and try to get it down in a clean way, probably shooting for 90 - 95 characters per line to start.

**Comment by gregvenech** [comment]
Closed via 46dbf283. Taking the screen size down 1024 still leaves the sidebar with a decent amount of room and yields around 95 characters per line.
